x-pack/plugins/ml/server/models/job_audit_messages/job_audit_messages.d.ts

```diff
@@ -21,7 +22,7 @@ export function jobAuditMessagesProvider(
       start?: string;
       end?: string;
     }
-  ) => any;
+  ) => { messages: JobMessage[]; notificationIndices: string[] };
   getAuditMessagesSummary: (jobIds?: string[]) => any;
-  clearJobAuditMessages: (jobId: string) => any;
+  clearJobAuditMessages: (jobId: string, notificationIndices: string[]) => any;
```

Comment:
should it be `{ success: boolean, last_cleared: number }` instead of `any`?
```suggestion
  clearJobAuditMessages: (jobId: string, notificationIndices: string[]) => { success: boolean, last_cleared: number };
```